geonode/qgis_server/tests/test_views.py

```diff
@@ -325,6 +325,24 @@ def test_ogc_specific_layer(self):
         uploaded.delete()
         vector_layer.delete()
 
+    @on_ogc_backend(qgis_server.BACKEND_PACKAGE)
+    def test_upload_geojson(self):
+        """Test we can upload a geojson layer and check bbox"""
+        file_path = os.path.realpath(__file__)
+        test_dir = os.path.dirname(file_path)
+        # a geojson sample is taken from http://geojson.org
+        geojson_file_path = os.path.join(test_dir, 'data', 'point.geojson')
+
+        uploaded = file_upload(geojson_file_path)
+        # check if file is uploaded
+        self.assertTrue(uploaded)
+        # check if bbox exist
+        expected_bbox = '125.599999999999994,' \
+                        '10.100000000000000,' \
+                        '125.599999999999994,' \
+                        '10.100000000000000'
+        self.assertEqual(uploaded.bbox_string, expected_bbox)
```

Comment:
Use `self.assertAlmostEqual` so you can round it to 2 or 1 decimal places :).

Reply:
I think travis doesn't pass because of that, so if you use assertAlmostEqual, I think it will pass.